packages/react-components/react-jsx-runtime/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-jsx-runtime",
   "version": "9.0.0-alpha.16",
-  "description": "React components for building web experiences",
+  "description": "Custom v9 runtime for JSX",
```

```suggestion
  "description": "Custom JSX runtime for @fluentui/react-components",
```

Let's not mention v9, after a version bump it will not make sense